src/molecules/dropdownButton.jsx

```diff
@@ -0,0 +1,107 @@
+import ClassNames from 'classnames';
+import PropTypes from 'prop-types';
+import React, { useRef, useState } from 'react';
+import { buttonPropTypes, buttonDefaultProps } from '../atoms/buttonConstants';
+import Button from '../atoms/button';
+import DropdownMenu from '../atoms/dropdownMenu';
+import DropdownMenuOption from '../atoms/dropdownMenuOption';
+import Icon from '../atoms/icon';
+
+const propTypes = {
+    ...buttonPropTypes,
+    as: PropTypes.oneOf(['button']),
```

Comment:
Why keep this is a `oneOf` (or even have it at all) if the only valid value is "button"?

Reply:
You're right.

Reply:
fixed.